src/Illuminate/Foundation/Auth/Access/Authorizable.php

```diff
@@ -18,6 +18,18 @@ public function can($abilities, $arguments = [])
         return app(Gate::class)->forUser($this)->check($abilities, $arguments);
     }
 
+    /**
+     * Determine if the entity has any of the given abilities.
+     *
+     * @param  iterable|string  $abilities
+     * @param  array|mixed  $arguments
+     * @return bool
+     */
+    public function canany($abilities, $arguments = [])
```

Comment:
`canAny`?

Reply:
Makes sense but I'd keep same syntax with blade and other methods (can, cannot). Final thoughts?

Reply:
Simliar to spatie's Permission package, I would go with `canAny` in terms of the method name, and lowercase `canany` for the blade directives.